analyzer/src/funTest/kotlin/AnalyzerFunTest.kt

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C) 2025 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.analyzer
+
+import io.kotest.core.spec.style.WordSpec
+import io.kotest.matchers.collections.containExactly
+import io.kotest.matchers.should
+
+import java.io.File
+import java.io.IOException
+
+import org.ossreviewtoolkit.analyzer.Analyzer.ManagedFileInfo
+import org.ossreviewtoolkit.model.ProjectAnalyzerResult
+import org.ossreviewtoolkit.model.config.AnalyzerConfiguration
+import org.ossreviewtoolkit.model.config.RepositoryConfiguration
+
+class AnalyzerFunTest : WordSpec({
```

Comment:
Could this be just a Test instead of a FunTest?

I'm asking also, because I have a hunch that the Dummy package manager in this
case might be simplify-able by replacing it with a mock.

Reply:
I basically agree to both, the dummy package manager could be a mock instead, and (at least) then the "FunTest" should be a "Test". However, as the excessive use of mocks has recently created some discussion in ORT Server context, I deliberately tried to avoid mocking (and a dependency on MockK) here, as the test can easily be implemented without mocking, and I don't see that mocking would make the test more readable.

So, while I'd be fine with renaming the "FunTest" to "just "Test", I'd prefer to not introduce any mocking.

Reply:
> I don't see that mocking would make the test more readable.

I gave it a quick try, and soon enough reached a point where the number of answers that I need to configure for the mock makes things less readable.

Reply:
I think turning it into non-fun-test would still make sense, but I'm ok if we keep the fun if you prefer.

Reply:
As I don't mind, I've moved it to `analyzer/src/test/kotlin/AnalyzerTest.kt` now.